packaging/debian/package_tool/package_tool

```diff
@@ -3,41 +3,25 @@
 # Copyright (c) .NET Foundation and contributors. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-
-# Takes Parameters:
-#   $1 = Input Directory
-#   $2 = Output Directory
-#   $3 = Package Version
-
 set -e
-
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
can you instead source `_common.sh` and make the path REPOROOT-relative? it makes moving files much easier...